fables/errors.py

```diff
@@ -29,3 +29,8 @@ class ParseError(Error):
 @dataclass
 class ExtractError(Error):
     pass
+
+
+@dataclass
+class InsufficientDetectorConfidenceError(Error):
```

Comment:
[non-blocking] I wonder if `InsufficientEncodingDetectorConfidenceError` might be a better, if longer, name for this error.